asm/benchRptAxpy.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
This code seems more complicated than necessary. Can we drop it and the .gitignore?